ui/src/main/webapp/src/app/components/project-list/project-list.component.ts

```diff
@@ -56,7 +56,7 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     ngOnInit():any {
         this.updateSort();
         this.getMigrationProjects();
-        this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
+        //this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
```

Comment:
I've commented this out as it always replaced the DOM I was inspecting and modifying.
However does it make sense to query projects every 30 seconds? How often do we expect that to change?
Could we perhaps try some push way?